evennia/accounts/accounts.py

```diff
@@ -870,10 +925,10 @@ def create(cls, *args, **kwargs):
 
             if account and _AUTO_CREATE_CHARACTER_WITH_ACCOUNT:
                 # Auto-create a character to go with this account
-
-                character, errs = account.create_character(
-                    typeclass=kwargs.get("character_typeclass")
-                )
+                call_kwargs = {}
+                if "character_typeclass" in kwargs:
+                    call_kwargs["character_typeclass"] = kwargs["character_typeclass"]
+                character, errs = account.create_character(**call_kwargs)
```

Comment:
I don't think `account.create_character` makes use of a kwarg `character_typeclass` ... it expects `typeclass`.